.github/workflows/roll20-update-benchmark.yml

```diff
@@ -0,0 +1,63 @@
+name: Publish benchmarks to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 */14 * *'  # every 14 days at midnight UTC
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+env:
+  BENCHMARKS_URL: https://roll20.github.io/detect-gpu
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+
+      - name: Install & build
+        run: |
+          npm install --legacy-peer-deps
+          npm run build
+
+      - name: Update benchmarks
+        run: npm run update-benchmarks
+
+      - name: Validate *local* benchmark JSON shape
```

Comment:
This is where we run the new test to verify that the json generated from the https://gfxbench.com/result.jsp matches the shape we expect